src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -51,9 +51,6 @@
 namespace osvr {
 namespace pluginhost {
     static const auto PLUGIN_HOST_LOGGER_NAME = "PluginHost";
-#ifdef _MSC_VER
```

Comment:
If the `.debug` suffix is no longer MSVC-only, then you should also modify `cmake-local/osvrAddPlugin.cmake` accordingly (lines 45–48).